dom.bs

```diff
@@ -6727,11 +6757,34 @@ are:
 <p>The <dfn method for=Element><code>attachShadow(<var>init</var>)</code></dfn> method steps are:
 
 <ol>
- <li><p>If <a>this</a>'s <a for=Element>namespace</a> is not the <a>HTML namespace</a>,
+ <li><p>Run <a>attach a shadow root</a> with <a>this</a>, <var>init</var>'s
+ {{ShadowRootInit/mode}}, <var>init</var>'s {{ShadowRootInit/clonable}}, and <var>init</var>'s
```

Comment:
It needs to be `<var>init<var>["{{ShadowRootInit/mode}}"]` when you reference members from a dictionary (map syntax).

Reply:
Done.